### PR TITLE
Doc: wow, what an improvment! But i will reorganize everything

### PR DESCRIPTION
**Pull request by stifoon**

### CODE REVIEW

File: documentation/slate/source/includes/interface.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
+The only endpoint of this version of the api is :
+<https://api.navitia.io/v1/>
+
 We aim to implement [HATEOAS](http://en.wikipedia.org/wiki/HATEOAS)
 concept with Navitia.
 
-Each response contains a linkable object and lots of links. Links allow
+All the resources return a response containing a links object, a paging
+object, and the requested object. That's lots of links. Links allow
```
Comment (antoine-de) [inline documentation/slate/source/includes/interface.md:11]: and the requested object**s**

File: documentation/slate/source/includes/isochrones.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<a name="isochrones"></a>Isochrones
+===================================
+
+-   **[Journeys](#journeys)** : Compute journeys
```
Comment (antoine-de) [inline documentation/slate/source/includes/isochrones.md:4]: isn't it a bit light ? but maybe you plan to add more stuff here later

File: documentation/slate/source/includes/nearby.md [hunk 3]
```diff
@@ -25,32 +19,17 @@ object, and the requested object.
...
--   **[Journeys](#journeys)** : Compute journeys
-
-| url | Result |
-|------------------------------------------|-------------------------------------|
-| `get` /coverage/*resource_path*/journeys | List of journeys or isochrones      |
-| `get` /journeys                          | List of journeys or isochrones      |
-
-
--   **[Route Schedules](#route-schedules)**, **[Stop Schedules](#stop-schedules)**, **[Departures](#departures)** and **[Arrivals](#arrivals)** : 
+-   **[Stop Schedules](#stop-schedules)**, **[Departures](#departures)** and **[Arrivals](#arrivals)** :
```
Comment (pbougue) [inline documentation/slate/source/includes/nearby.md:22]: This is now duplicated, no?

File: documentation/slate/source/includes/timetables.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<a name="timetables"></a>Timetables
+=================================
+
+-   **[Route Schedules](#route-schedules)** and **[Stop Schedules](#stop-schedules)** :
```
Comment (pbougue) [inline documentation/slate/source/includes/timetables.md:4]: Stop schedules is duplicated, then?

File: documentation/slate/source/includes/nearby.md [hunk 5]
```diff
@@ -25,32 +19,17 @@ object, and the requested object.
...
-
--   **[Route Schedules](#route-schedules)**, **[Stop Schedules](#stop-schedules)**, **[Departures](#departures)** and **[Arrivals](#arrivals)** : 
+-   **[Stop Schedules](#stop-schedules)**, **[Departures](#departures)** and **[Arrivals](#arrivals)** : 
 Compute time tables for a given resource
 
 | url | Result |
 |-------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
-| `get` /coverage/*resource_path*/route_schedules | List of the entire route schedules for a given resource                                                       |
 | `get` /coverage/*resource_path*/stop_schedules  | List of the stop schedules grouped by ``stop_point/route`` for a given resource                               |
+| `get` /coverage/*lon;lat*/stop_schedules        | List of the stop schedules grouped by ``stop_point/route`` for coordinates                                    |
```
Comment (pbougue) [inline documentation/slate/source/includes/nearby.md:28]: some more precisions on the differences with the previous line would be nice (also for lines 30 and 32)

**Pull request merged**